src/progress.rs

```diff
@@ -421,7 +431,7 @@ impl ProgressSet {
 
     /// Promote a learner to a peer.
     pub fn promote_learner(&mut self, id: u64) -> Result<()> {
-        debug!("Promote learner with id {}.", id);
+        info!(self.logger, "Promoting peer with id {id}", id = id);
```

Comment:
Quite a few of these change from `info` to `debug`, that might have been stuff from #52